test/conftest.py

```diff
@@ -360,83 +379,85 @@ def inner(revs, bugs, message=None, bookmarks="mozilla/autoland"):
         if message is None:
             message = b"\n".join(msg)
         return hg_commit(hg_gecko_upstream, message, bookmarks)
+
     return inner
 
 
 @pytest.fixture
 def gecko_worktree(env, git_gecko):
-    path = os.path.join(env.config["root"],
-                        env.config["paths"]["worktrees"],
-                        "gecko"
```

Comment:
not sure if the comma was supposed to be here between "gecko" and "autoland"? so it wouldn't be merged into `geckoautoland`